### PR TITLE
Add Player class

### PR DESCRIPTION
**Pull request by nic3-14159**
Add a new class modelling a player of the game, with basic attributes and getters/setters.

### CODE REVIEW

File: app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     */
+    private ArrayList<QRCode> codes;
+
+
+    public Player(String username, String phoneNumber) {
+        this.username = username;
+        this.phoneNumber = phoneNumber;
+    }
+
+    public void setUsername(String username) {
```
Comment (real-rookie) [inline app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java:28]: For now, we do not allow users to changes their names

File: app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        this.username = username;
+    }
```
Comment (real-rookie) [inline app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java]: For now, we do not allow users to change their usernames. We use usernames to uniquely identify users.

**Review by real-rookie — CHANGES_REQUESTED** [review]
Should also have a method for adding a QR code to the user's list

File: app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java (hunk 2 above)
Comment (nic3-14159) [reply app/src/main/java/com/cmput301w23t40/capturetheqr/Player.java]: Done

**Comment by nic3-14159** [comment]
> Should also have a method for adding a QR code to the user's list

Done

**Review by real-rookie — APPROVED** [review]
do not forget to address issue #32

**Pull request merged**